### ISSUE TITLE
Installing does not work on python 3.9

### ISSUE THREAD

**Issue by Filco306** [ray-project/tune-sklearn#169]
Hello, 

I currently cannot install this package when I use the pip installment. 

I have tried both ways of installing, and I get: 

```bash
pip3 install tune-sklearn ray[tune]
ERROR: Could not find a version that satisfies the requirement ray[tune]
ERROR: No matching distribution found for ray[tune]
```

```bash
pip3 install -U git+https://github.com/ray-project/tune-sklearn.git && pip3 install 'ray[tune]'
Collecting git+https://github.com/ray-project/tune-sklearn.git
  Cloning https://github.com/ray-project/tune-sklearn.git to /private/var/folders/g4/2rh0j8g948b6vps252ckmk1w0000gn/T/pip-req-build-lfi_20kv
Requirement already satisfied: scipy in /usr/local/lib/python3.9/site-packages (from tune-sklearn==0.2.0) (1.5.4)
ERROR: Could not find a version that satisfies the requirement ray[tune] (from tune-sklearn)
ERROR: No matching distribution found for ray[tune]
```

My python3 version: 

```python3
>> import sys
>> print(sys.version)
3.9.0 (default, Dec  6 2020, 18:02:34) 
[Clang 12.0.0 (clang-1200.0.32.27)]

```

Would you perhaps know what is wrong? Thank you very much!

**Comment by richardliaw** [comment]
Yeah! It's because one of the core dependencies (Ray) currently doesn't work on 3.9. I've brought this up with the team and there's currently ongoing work (expect to be available in January!)

**Comment by Filco306** [comment]
Okay awesome, thank you very much! :D

**Comment by Filco306** [comment]
Working now! Closing this!

**Comment by Miladiouss** [comment]
I still have this issue. pip install -U ray says ERROR: No matching distribution found for ray

**Comment by richardliaw** [comment]
What version of pip are you using @Miladiouss ?

**Comment by Miladiouss** [comment]
@richardliaw 
```
pip                       21.0.1           py39h06a4308_0  
python                    3.9.4                hdb3f193_0  
```

**Comment by Miladiouss** [comment]
Btw, I'm using Ubuntu 20.

**Comment by lindu2770** [comment]
same error here
pip 21.1.1
Python 3.8.8

**Comment by richardliaw** [comment]
@Miladiouss I'll just keep this open - we don't have 3.9 support for Ray yet. 

@lindu2770 not sure why you're seeing this error...

**Comment by Miladiouss** [comment]
Thanks @richardliaw, for reference for others, I had to go to Python 3.7 to install ray.

**Comment by LasithEranga** [comment]
Thanks @Miladiouss Python 3.7 worked for me as well.

**Comment by koaning** [comment]
When I go to https://pypi.org/project/ray/#history then I see the following versions supported:

<img width="288" alt="image" src="https://user-images.githubusercontent.com/1019791/169990286-9a1e9f97-40bc-456e-9205-79f561f6a1d7.png">

If 3.9 is not supported yet, maybe the version mentioned should be removed.

**Comment by hannahbus** [comment]
I encountered the same issue with Python 3.10, which is supported by Ray 1.12.1 according to the setup.py. The error was
_ERROR: Could not find a version that satisfies the requirement ray (from versions: none)
ERROR: No matching distribution found for ray_

**Comment by deifactor** [comment]
I *suspect* this is because ray doesn't publish sdists, only wheels, and everyone getting bitten by this is running ARM64 Linux, which Ray doesn't publish any wheels for.

**Comment by risboo6909** [comment]
@deifactor Thanks for the hint! Had the similar issue, fixed it by switching from aarch64 to amd64.

**Comment by 66Gramms** [comment]
This is still a problem on windows 10 on python 3.10

**Comment by AndreMPCosta** [comment]
> 

Can you give the link for the wheels? I can't find those.

**Comment by Vasiliy-b** [comment]
ERROR: Could not find a version that satisfies the requirement ray[default,tune]>=2.0.0 (from finrl) (from versions: none)
ERROR: No matching distribution found for ray[default,tune]>=2.0.0

Trying with py 3.9, 3.10.5 and few other. On win 11. 
Try as well directly pip installing .whl 
Nothing seems to help.

I am not experienced with python. fyi.

**Comment by salmanea** [comment]
On Python 3.11 and it does not install either.

**Comment by Gustinouz** [comment]
> > > For me, problem solved with directly pip installing .whl from https://docs.ray.io/en/latest/ray-overview/installation.html which I am py3.10 on win10, just for reference ![image](https://user-images.githubusercontent.com/95896950/192256803-1e68f6ea-be36-44b5-98dd-44f78c937c8b.png)
> > 
> > 
> > @GLGDLY
> > Can you give the link for the wheels? I can't find those.
> 
> linux： https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-3.0.0.dev0-cp310-cp310-manylinux2014_x86_64.whl
> 
> macos: https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-3.0.0.dev0-cp310-cp310-macosx_10_15_universal2.whl
> 
> Windows: https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-3.0.0.dev0-cp310-cp310-win_amd64.whl
> 
> just change the version in the url to get wheels for other python versions, such as: https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-3.0.0.dev0-cp39-cp39-win_amd64.whl

tried this but s

> > > For me, problem solved with directly pip installing .whl from https://docs.ray.io/en/latest/ray-overview/installation.html which I am py3.10 on win10, just for reference ![image](https://user-images.githubusercontent.com/95896950/192256803-1e68f6ea-be36-44b5-98dd-44f78c937c8b.png)
> > 
> > 
> > @GLGDLY
> > Can you give the link for the wheels? I can't find those.
> 
> linux： https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-3.0.0.dev0-cp310-cp310-manylinux2014_x86_64.whl
> 
> macos: https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-3.0.0.dev0-cp310-cp310-macosx_10_15_universal2.whl
> 
> Windows: https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-3.0.0.dev0-cp310-cp310-win_amd64.whl
> 
> just change the version in the url to get wheels for other python versions, such as: https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-3.0.0.dev0-cp39-cp39-win_amd64.whl

tried this but still have errors
ERROR: Could not find a version that satisfies the requirement ray-3.0.0.dev0-cp310-cp310-win_amd64 (from versions: none)
ERROR: No matching distribution found for ray-3.0.0.dev0-cp310-cp310-win_amd64

**Comment by Gustinouz** [comment]
> > > For me, problem solved with directly pip installing .whl from https://docs.ray.io/en/latest/ray-overview/installation.html which I am py3.10 on win10, just for reference ![image](https://user-images.githubusercontent.com/95896950/192256803-1e68f6ea-be36-44b5-98dd-44f78c937c8b.png)
> > 
> > 
> > @GLGDLY
> > Can you give the link for the wheels? I can't find those.
> 
> linux： https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-3.0.0.dev0-cp310-cp310-manylinux2014_x86_64.whl
> 
> macos: https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-3.0.0.dev0-cp310-cp310-macosx_10_15_universal2.whl
> 
> Windows: https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-3.0.0.dev0-cp310-cp310-win_amd64.whl
> 
> just change the version in the url to get wheels for other python versions, such as: https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-3.0.0.dev0-cp39-cp39-win_amd64.whl

tried this but still have errors
ERROR: Could not find a version that satisfies the requirement ray-3.0.0.dev0-cp310-cp310-win_amd64 (from versions: none)
ERROR: No matching distribution found for ray-3.0.0.dev0-cp310-cp310-win_amd64

**Comment by dss010101** [comment]
well..im getting this error today trying to rebuild my container...
